### PR TITLE
COECP-345 - allow upsert in patch requests.

### PR DESCRIPTION
**Pull request by swilam**

### CODE REVIEW

File: src/ET_Client.php [hunk 1]
```diff
@@ -249,7 +249,7 @@ function __construct($getWSDL = false, $debugSoap = false, $params = null)
      * @return void
      * @throws Exception
      */
-    function refreshToken($forceRefresh = false)
+    function refreshToken($forceRefresh = true)
```
Comment (phpguru) [inline src/ET_Client.php:252]: This can take a lot of time. Their wsdl is huge.

**Review by phpguru — COMMENTED** [review]
This looks good, I haven't tested it, but I am wondering about forcing the wsdl to download everytime, that's my only concern.

**Comment by swilam** [comment]
@phpguru Thanks a lot for your review! 
Regarding the forceRefresh, I would like to run a background worker that is consuming messages from a queue and exporting these as data extensions to Salesforce. So in this particular case i need the worker to be always-on, and without refreshing the token i end up with the problem that the token expires, and the worker stops.

**Comment by phpguru** [comment]
@swilam oh! I misread that. Elsewhere there’s a force refresh of the wdsl, this is just the token. Got it.

**Pull request merged**